charts/airbyte/Chart.yaml

```diff
@@ -37,3 +38,27 @@ dependencies:
     name: minio
     version: 11.x.x
     repository: https://charts.bitnami.com/bitnami
+  - condition: airbyte-bootloader.enabled
+    name: airbyte-bootloader
+    repository: "https://airbytehq.github.io/helm-charts/"
+    version: placeholder
+  - condition: temporal.enabled
+    name: temporal
+    repository: "https://airbytehq.github.io/helm-charts/"
+    version: placeholder
```

Comment:
Are these `placeholders` supposed to be here @xpuska513 ? I just submitted an issue here: https://github.com/airbytehq/airbyte/issues/15656

Reply:
They're supposed, let me update the docs and explained a little bit more in the #15656